packages/database/src/modelClasses/Entity.js

```diff
@@ -131,6 +137,26 @@ class EntityType extends DatabaseType {
 
     throw new Error(`Maximum of (${MAX_ENTITY_HIERARCHY_LEVELS}) entity hierarchy levels reached`);
   }
+
+  async getChildren(hierarchyId) {
+    //Raw sql:
+    //     SELECT entity.*
+    //     FROM entity
+    //     INNER JOIN entity_relation on entity.id = entity_relation.child_id
+    //     WHERE entity_relation.parent_id = `'${this.id}'`
+    //     AND entity_relation.entity_hierarchy_id = `'${hierarchyId}'`;
+    return this.model.find(
+      {
+        [`${TYPES.ENTITY_RELATION}.parent_id`]: this.id,
+        [`${TYPES.ENTITY_RELATION}.entity_hierarchy_id`]: hierarchyId,
+      },
+      {
+        joinWith: TYPES.ENTITY_RELATION,
+        joinType: JOIN_TYPES.INNER,
+        joinCondition: [`${TYPES.ENTITY}.id`, `${TYPES.ENTITY_RELATION}.child_id`],
+      },
+    );
+  }
```

Comment:
To check project level visualisations (eg: UNFPA regional mapOverlay/dashboardReports,...), I need to check the child countries of the project entity and check if the user has access to any of those countries. However, the logic for project hierarchies is only in `web-config-server`, not in `meditrak-server`. Is this an acceptable solution for now? It just seems too much work and out of scope to bring the same project hierarchies logic from `web-config-server` to here just for this task... Maybe we can work on it as a core product task later